data/udisks2.service.in

```diff
@@ -3,10 +3,27 @@ Description=Disk Manager
 Documentation=man:udisks(8)
 
 [Service]
-Type=dbus
 BusName=org.freedesktop.UDisks2
+CapabilityBoundingSet=CAP_SYS_RAWIO
 ExecStart=@udisksdprivdir@/udisksd
+IPAddressDeny=any
 KillSignal=SIGINT
+LimitMEMLOCK=0
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProtectHome=yes
```

Comment:
We're spawning commands on behalf of different UID. Not sure how this works exactly, certain commands may need access to homedir. So if there's a danger this limitation would be applied to spawned children, this should be removed then.

Reply:
It's applied to child processes, yes. I'll remove both.